homeassistant/components/zha/cover.py

```diff
@@ -278,3 +280,31 @@ async def async_stop_cover(self, **kwargs) -> None:
         if not isinstance(res, list) or res[1] != Status.SUCCESS:
             self.debug("couldn't stop cover: %s", res)
             return
+
+
+@STRICT_MATCH(
+    channel_names={CHANNEL_LEVEL, CHANNEL_ON_OFF}, manufacturers="Keen Home Inc"
+)
+class KeenVent(Shade):
+    """Keen vent cover."""
+
+    @property
+    def device_class(self) -> Optional[str]:
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        return DEVICE_CLASS_DAMPER
+
+    async def async_open_cover(self, **kwargs):
+        """Open the window cover."""
```

Comment:
Vent cover?

Reply:
Argh, it is going to be just cover then :)